src/main/webapp/app/exercises/shared/example-submission/example-submission-import/example-submission-import.component.ts

```diff
@@ -63,6 +64,11 @@ export class ExampleSubmissionImportComponent implements OnInit {
                 this.content = resp;
                 this.loading = false;
                 this.total = resp.numberOfPages * this.state.pageSize;
+                if (this.content?.resultsOnPage?.length > 0) {
+                    this.content.resultsOnPage.forEach((submission) => {
+                        submission.submissionSize = this.exampleSubmissionService.getSubmissionSize(submission, this.exercise);
+                    });
+                }
```

Comment:
[nit] Isn't the check for length redundant? I think you can just call:
`this.content?.resultsOnPage?.forEach((submission) => ...);`

Reply:
wanted to be sure there is element in the array, but I can check...